SwitchDemo/SwitchDemo/Activity1.cs

```diff
@@ -0,0 +1,23 @@
+namespace SwitchDemo
+{
+    [Activity(Label = "SwitchDemo", MainLauncher = true)]
+    public class Activity1 : Activity
+    {
+        protected override void OnCreate(Bundle? bundle)
+        {
+            base.OnCreate(bundle);
+
+            SetContentView(Resource.Layout.Main);   
+
+            var s = FindViewById<Switch>(Resource.Id.monitored_switch);
+
+            ArgumentNullException.ThrowIfNull(s);
+            s.CheckedChange += delegate (object? sender, CompoundButton.CheckedChangeEventArgs e) {
```

Comment:
Can you use the newer C# style, like:
```suggestion
            s.CheckedChange += (sender, e) => {
```

Reply:
Hi Jonathan! I just updated the file